src/services/pathCompletions.ts

```diff
@@ -312,9 +312,11 @@ namespace ts.Completions.PathCompletions {
 
     function getCompletionEntriesFromTypings(host: LanguageServiceHost, options: CompilerOptions, scriptPath: string, span: TextSpan, result: CompletionEntry[] = []): CompletionEntry[] {
         // Check for typings specified in compiler options
+        const seen = createMap<true>();

```

Comment:
Could you add a test that fails without this?

Reply:
Oops, I meant to.